### ISSUE TITLE
Windows VS2022 installation

### ISSUE THREAD

**Issue by zchrissirhcz** [glmcdona/binary2strings#14]
Hi, @glmcdona 

Just find this repo interesting and would like to try, as alternative to strings.exe on Windows.

## 1. pip failed
- System: Windows 10
- Python: Python 3.10, installed via miniconda
- C++ Compiler / IDE: I have Visual Studio 2019, 2022 both installed.

The output: it complains cl.exe failed.

## 2. C++ source modification suggestion
https://github.com/glmcdona/binary2strings/blob/main/src/extracted_string.cpp#L11

This line11 in extracted_string.cpp, convert NULL to std::string. However it will cause crash if you just try the minimal example on linux-x64 with clang:
```c++
#include <stdio.h>
#include <string.h>
int main()
{
    std::string m = (std::string)NULL;
    std::cout << "m is " << m << std::endl;
    return 0;
}
```

## 3. Build source and install on Windows
OK, based on the above two considerations:
- I commented out line 11 in extracted_string.cpp
- In an opened cmd.exe, `python setup.py install` still fail
- Open VS2022 x64 native command prompt, then `python setup.py install` success.

I would suggest fix the `pip install binary2strings` experience on Windows. Thank you.

**Comment by glmcdona** [comment]
Thanks for your report, I'll look into it! I've had success with it on Windows 10 here, so like you suggested it may depend on build tools that are installed. I'll see if I can repro and make it compile by default regardless of build tools installed.

**Comment by glmcdona** [comment]
Fixed with #16. Apologies for the delay! Thanks for identifying the issue @zchrissirhcz .

**Comment by glmcdona** [comment]
Fixed with release 0.1.10. You should be able to just do `pip install binary2strings` now.
